server/src/test/java/io/spine/server/storage/AbstractRecordStorageTest.java

```diff
@@ -66,10 +66,12 @@
 /**
  * Abstract base for tests of record storages.
  *
- * @param <I> the type of identifiers a storage uses
- * @param <S> the type of storage under the test
+ * @param <I>
+ *         the type of identifiers a storage uses
+ * @param <S>
+ *         the type of storage under the test
  */
-public abstract class AbstractRecordStorateTest<I, S extends RecordStorage<I>>
+public abstract class AbstractRecordStorageTest<I, S extends RecordStorage<I>>
```

Comment:
Please have a description in the Javadoc on why this test class has no nested classes.